metrics/cartridge/clock.lua

```diff
@@ -1,5 +1,11 @@
 local utils = require('metrics.utils')
-local membership = require('membership')
+local ok, membership = pcall(require, 'membership')
+
+if not ok then
+    return { update = function() end }
```

Comment:
setting default metrics will result in "attempt to index nil value 'list'" error

Reply:
The fix is suggested below in `delete_collectors` function